Gpt checker part 2 (#19)
greenelab/word_lapse_manuscript: content/05.discussion.md

Language is rapidly evolving, and the usage of words changes over time, with words assimilating new meanings or associations [@doi:10.1093/acrefore/9780199384655.013.323].
Some efforts have been made to study semantic change using biomedical text [@doi:10.1142/9789811232701_0011;@arxiv:2102.07836; @doi:10.2196/22635]; however, no such work has examined the changes evident in both pre-publication peer-reviewed and preprinted biomedical text.

> We examined semantic changes in open-access biomedical corpora,Pubmed (PMCOA), and bioRxiv/MedRxiv,  over a two-decade period from 2000 to 2022.

Reviewer: ```suggestion
We examined semantic changes in two open-access biomedical corpora, PubMed (PMCOA) and bioRxiv/MedRxiv, over a two-decade period from 2000 to 2022.
```

> We developed a novel statistic that incorporated multiple Word2Vec models to examin semantic change over time.

Reviewer: ```suggestion
We developed a novel statistic that incorporated multiple Word2Vec models to examine semantic change over time.
```

> We used orthogonal processes to align each model, and we found that the word vectors were closer together after alignment (Figure {@fig:word2vec_alignment}).

Reviewer: ```suggestion
We used orthogonal procrustes to align each model, and we found that the word vectors were closer together after alignment (Figure {@fig:word2vec_alignment}).
```

However, the best approach to align these models still remains to be determined [@doi:10.1007/978-3-030-32233-5_58].
As has been reported in previous studies [@doi:10.1162/tacl_a_00008; @doi:10.48550/arXiv.1804.09692], we found that without a correction step for the variability within and across years, it is difficult to compare stable and unstable models.
Our correction approach revealed that the average distances in the earlier years had less variability when using multiple models than when using a single model (Figure {@fig:novel_distance_validation}).

After correcting for year variability, our analysis revealed more than 41,000 change points, including tokens such as 'cas9', 'pandemic', and 'sars' (Figure {@fig:preprint_published_changepoints}).
Many of these change points overlapped between PMCOA and preprints, and were related to COVID-19 (Table {@tbl:published_preprint_change_table}).

> This indicates that the COVID-19 pandemic has been strong and immediate enough to cause rapid semantic change across both publishing paradigms [@doi:10.1371/journal.pbio.3000959; @doi:10.1371/journal.pone.0240123].

Reviewer: ```suggestion
This indicates that the COVID-19 pandemic has had a sufficiently strong impact on the biomedical literature to cause rapid semantic change across both publishing paradigms [@doi:10.1371/journal.pbio.3000959; @doi:10.1371/journal.pone.0240123].
```
